From ade93a31be1ffd204d2a1337df4fb376d4e4544d Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 12 Aug 2018 11:50:07 +0300
Subject: [PATCH] btl: pcie: enable the GEN3 EQ REDO

Enable the GEN3 EQ REDO.
More precisely, reset to default.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/btl/pcie_phy.s b/board/habanalabs/common/btl/pcie_phy.s
index 5e96b13..5e3086e 100644
--- a/board/habanalabs/common/btl/pcie_phy.s
+++ b/board/habanalabs/common/btl/pcie_phy.s
@@ -37,9 +37,12 @@
 cfg mmPCIE_PHY_PROTOCOL0_EXT_TX_VBOOST_LVL, 0x00000004
 cfg mmPCIE_PHY_PHY_EXT_CTRL_SEL, 0x00000001
 
-/* disable the EQ REDO */
-/* NOTE:::::::::::::::::: this is BAD do not release with this */
-cfg mmPCIE_DBI_GEN3_RELATED_OFF, 0x00402801
+/*
+ * disable the EQ REDO
+ * Currently, we are not certain this is good or bad...
+ * Meanwhile leave this out.
+ */
+/* cfg mmPCIE_DBI_GEN3_RELATED_OFF, 0x00402801 */
 
 /*
  * enable direction of change,
